masterlist.yaml

```diff
@@ -301,3 +305,6 @@ plugins:
       - <<: *patchAvailable
         subs: [ 'DarkerNights.esp' ]
         condition: 'active("Radiant Clouds and Fogs.esp") and active("DarkerNights.esp") and not active("DarkerNights-Radiant.esp")'
+  - name: 'Scrap Everything.esp'
+    msg: [ *doNotClean ]
```

Comment:
I looked over the mod description, and it doesn't mention this anywhere. What's your source for this?